### PR TITLE
Create Profile/Home Route and View

### LINKED ISSUE #42: feat: add home route & view

### LINKED ISSUE #43: feat: add user profile route & view

### PR DESCRIPTION
**Pull request by jksmall0631**
This PR closes #42 closes #43 

#### What does this PR do?
-Create route and view for Home
-Create route and view for User

### CODE REVIEW

File: src/LoginAuth0.js [hunk 1]
```diff
@@ -68,7 +68,8 @@ class LoginAuth0 extends Component {
               }
             });
           });
-        } else {
+        }
+        else {
```
Comment (dviramontes) [inline src/LoginAuth0.js]: can we do this in one line ? just a style thing

**Review by dviramontes — APPROVED** [review]
other than the one comment, lgtm! thanks @jksmall0631 ! !

File: src/LoginAuth0.js (hunk 1 above)
Comment (jksmall0631) [reply src/LoginAuth0.js]: You mean with a ternary operator or just like it was?

Comment (dviramontes) [reply src/LoginAuth0.js]: no sorry,  i meant
```
} else {
```
instead of 
```
} 
else {
```

Comment (dviramontes) [reply src/LoginAuth0.js]: no big deal though

**Pull request merged**